style/middleware/_middleware.header.scss

```diff
@@ -0,0 +1,40 @@
+/*
+ ** TO-DO: Few bits with this comment to be removed once implemented in Core Nightingale
```

Comment:
Have we got issues raised in the repo to revisit these items?

Reply:
Yes @cehwitham [tkt-00024](https://github.com/NHSLeadership/moodle-theme_nightingale/issues/24)